
\begin{titlepage}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newcommand{\HRule}{\rule{\linewidth}{0.5mm}}

\center 

%logo
\includegraphics[width=7cm]{Figures/misc/ucllogo.jpg}\\[1cm]
%headings
\textsc{\LARGE University College London}\\[2cm] % Name of your university/college
%title
\HRule \\[1cm]
%CHANGE TITLE?
{ \huge \bfseries RNA dysregulation in models of frontotemporal dementia and amyotrophic lateral sclerosis}\\[0.4cm] 
\HRule \\[2cm]


\textsc{\Large Doctoral Thesis}\\[1.5cm] % Minor heading such as course title


%Author
%\begin{minipage}[t]{0.8\textwidth}
%\begin{flushleft} \large
%\raggedright
\emph{Author:}\\
Jack \textsc{Humphrey}\\[1cm] % Author's Name
%\end{flushleft}
%\end{minipage}

%\begin{minipage}[t]{0.8\textwidth}
%\begin{flushright} \large
%\raggedleft
\emph{Supervisors:} \\
Prof Adrian \textsc{Isaacs}\\ 
Dr Vincent \textsc{Plagnol}\\ 
Dr Pietro \textsc{Fratta}\\[1cm]


\textsc{\Large UCL Institute of Neurology}\\[0.5cm] % Major heading such as course name
\textsc{\Large UCL Genetics Institute}\\[2cm] % Major heading such as course name


%\end{flushright}
%\end{minipage}\\[3cm]

%date
%{\large \today}\\[3cm] 

\vfill % Fill the rest of the page with whitespace

\end{titlepage}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\cleardoublepage
%\clearpage

%\phantomsection
% DON"T FORGET TO SIGN THIS

I confirm that the work presented in this thesis is my own.  Where information has been derived from other sources, I confirm that this has been indicated in the thesis.

Jack \textsc{Humphrey}
\vfill


\cleardoublepage


%\hspace{0pt}
\vfill
%\phantomsection
%\begin{abstract}
\section*{Abstract}
%% update! 300 words max!
% 271 words
Amyotrophic Lateral Sclerosis (ALS) and Frontotemporal dementia (FTD) are two rare but devastating neurodegenerative diseases that share pathological features and genetic factors. 
A central question in both diseases is the role of the RNA-binding proteins transactive response DNA-binding protein 43kDa (TDP-43) and fused in sarcoma (FUS). 
These proteins play a vital role in RNA regulation in all cells but in diseased neurons they alter their cellular localisation to form potentially pathogenic aggregates. 
This process can be linked to rare genetic mutations in the \textit{TARDBP} and \textit{FUS} genes, although most cases of ALS and FTD have no known genetic cause. 

My work uses the revolutionary technology of RNA sequencing to measure and compare gene expression and RNA splicing in different cellular and animal models of sporadic and genetic disease.
Here I present the results of four studies that investigate the biology of TDP-43 and FUS, assessing both their normal cellular roles and the impact of rare disease-causing mutations.

In these projects I analyse RNA sequencing data to discover novel gene expression and RNA splicing phenomena.
This includes the repression of cryptic splicing by TDP-43 but not FUS, the progressive downregulation of mitochondrial and ribosomal transcripts in a mouse model of FUS ALS, a gain of splicing function by TDP-43 mutations affecting constitutive exon splicing, and widespread changes in intron retention caused by FUS knockout or aggressive FUS mutations. I also discover a novel mechanism for how FUS might regulate its own translation.

This work expands on what is currently known about the roles in RNA regulation for TDP-43 and FUS and provides new avenues for understanding both the causes and progression of ALS and FTD.

\hspace{0pt}
\vfill

%
%
%
%Amyotrophic Lateral Sclerosis (ALS) and Frontotemporal dementia (FTD) are two rare but devastating neurodegenerative diseases that share pathological features and genetic factors. 
%A central question in both diseases is the role of RNA-binding proteins, particularly transactive response DNA-binding protein 43kDa (TDP-43) and fused in sarcoma (FUS). 
%These proteins play a vital role in RNA regulation in all cells but in diseased neurons they alter their cellular localisation to form potentially pathogenic aggregates. 
%This process can be linked to rare genetic mutations in the \textit{TARDBP} and \textit{FUS} genes, although most cases of ALS/FTD have no known genetic cause. 
%My work uses the revolutionary technology of RNA sequencing (RNA-seq) to measure RNA expression and splicing in different cellular and animal models of sporadic and genetic disease.
%Here I present the results of four studies on the biology of TDP-43 and FUS, investigating both their normal cellular roles and the impact of rare disease-causing mutations.
%
%I re-analyse several public datasets of TDP-43 and FUS depletion and present tool to discovery and quantify novel splicing events.
%I establish that the repression of cryptic exons is a function of TDP-43 and not FUS.
% I describe work on a new mutant mouse model of FUS-mediated ALS. 
% I find a specific transcriptional signature in late adult spinal cords where mitochondrial and ribosomal genes are downregulated in the presence of a single copy of mutant FUS.
% I compare two TDP-43 mutant mouse lines.
% The first, a loss of RNA binding mutation, has widespread cryptic exon splicing equivalent to a loss of nuclear TDP-43.
% The second has a mutation in the TDP-43 low-complexity domain, the hotspot for ALS mutations.
% I observe the inverse phenomenon, the skipping of constitutive exons, suggesting a gain of splicing function.
% Finally, I compare three independent datasets where FUS is either knocked out or mutated in mouse neuronal cells.
% I present a joint modelling approach which boosts the power to detect gene expression and splicing changes.
% I observe a large overlap between knockout and mutation, suggesting that FUS mutations act to deplete nuclear FUS.
% When studying the FUS locus itself I observe a novel mechanism by which FUS could regulate its own translation.
% 
% 
%
%I conducted a re-analysis of published RNA sequencing experiments which deplete either TDP-43 or FUS as a crude model of sporadic disease. 
%I assessed the evidence for cryptic splicing, a novel RNA phenotype where previously silent sections of RNA are included into transcripts when a particular splicing factor is depleted. 
%I demonstrated that repression of cryptic splicing is a function specific to TDP-43 and not FUS. 
%I predicted that cryptic splicing disrupts mRNA stability and may therefore be a factor in cell death in ALS/FTD when TDP-43 is depleted from the nucleus.
%Later analysis of more RNA-binding proteins found evidence of cryptic splicing in nearly all of them, raising questions as to why they are not seen in FUS.
%
%I describe work on a new mutant mouse model of FUS-mediated ALS. 
%I assessed gene expression and splicing in two tissues and time points to look for progressive changes in RNA regulation by the mutant protein.
%I discovered a pattern of gene expression dysregulation occuring in late adult spinal cords only. 
%The affected genes were enriched in mitochondrial and ribosomal functions, suggesting a role for mutant FUS in regulating these two cellular functions. 
%
%% TDP mice
%I compared two TDP-43 mutant mice, chosen from a random mutagenesis screen to investigate TDP-43 function. 
%One line had a mutation in an RNA-recognition motif which reduced the ability of TDP-43 to bind mRNA.
%RNA-seq from these mice showed signs of TDP-43 loss of function, with widespread cryptic splicing.
%The other mouse line had a mutation in the TDP-43 low-complexity domain, where most ALS mutations are found. 
%This mutation appeared to cause a gain of splicing function characterised by an increase in exon skipping.
%These "skiptic exons" affect constitutive exons and their skipping led to frame shifting and transcript degradation through nonsense-mediated decay. 
%Some of these events can be seen in human patients with TARDBP mutations.
%
%
%Lastly, I compared three independent RNA-seq datasets where FUS was either knocked out or mutated in mouse neuronal cells.
%I analysed gene expression and splicing jointly for each condition, increasing the power to detect changes in RNA.
%The two conditions greatly overlap, suggesting that FUS mutations act to reduce the nuclear role of FUS.
%There was little evidence to support a gain of toxic function in the cytoplasm. 
%When I analysed the FUS locus itself I observed a specific change in FUS intron retention suggesting a novel mechanism of how FUS protein regulates its own translation.
%
%My goal is to transfer the insight gained from the model work into establishing mechanisms of disease course and aetiology in the analysis of human patient data. 






%\end{abstract}
%\clearpage

\cleardoublepage

\section*{Impact Statement}

% intro
% aims of thesis
% how these insights are useful to the field
This thesis comprises four studies on TDP-43 and FUS, two proteins linked to the devastating neurodegenerative diseases Amyotrophic Lateral Sclerosis and Frontotemporal Dementia.
I analysed RNA sequencing data to develop insights in to the functions of these two proteins in different models of disease.
This work has uncovered new roles for TDP-43 and FUS in specific types of RNA regulation.
Additionally, I have discovered new mechanisms to explain how disease-associated mutations in the two proteins affect these roles.
Conclusions from my work are applicable to the RNA biology field as a whole as well as efforts to understand and treat these diseases.
Insights into RNA-binding proteins and RNA regulation from this work can be transferred to many non-neurological diseases including muscular diseases, retinal diseases and certain cancers \citep{Scotti2015}. 
Several of my chapters focus on developing new methods or adapting existing methods to analyse RNA splicing, a key mechanism of RNA regulation.
RNA-sequencing is becoming a standard experiment across all fields of biology and the use of sequencing data in analysis of RNA splicing is an area of extreme interest and method development. 

% impacts arising from this thesis:
Three peer-reviewed papers have been published from this thesis.
As of November 2018, ``Quantitative analysis of cryptic splicing associated with TDP-43 depletion'' \citep{Humphrey2017} has been cited 11 times, ``Humanized mutant FUS drives progressive motor neuron degeneration without aggregation in FUS Delta14 knockin mice'' \citep{Devoy2017} has been cited 11 times and ``Mice with endogenous TDP-43 mutations exhibit gain of splicing function and characteristics of amyotrophic lateral sclerosis'' \citep{Fratta2018} has been cited 9 times and was the subject of a ``News and Views'' article in \textit{The EMBO Journal} \citep{Rouaux2018}.
These outcomes illustrate the importance of this work to the neurodegenerative disease and RNA biology fields.

Three new mouse models of ALS/FTD have been generated for projects carried out in this thesis: the FUS $\Delta$14 mouse, the TDP-43 RRM2mut mouse, and the TDP-43 LCDmut mouse. 
All three mice are available for labs around the world to use in further experiments.
RNA-seq data created from these mice have been made publicly available and can be downloaded from the Sequence Read Archive\footnotemark.
Several of my chapters rely on the re-analysis of published RNA sequencing data. 
I am pleased to see that sequencing data analysed in this thesis (chapter 5) has been reused in a further paper on TDP-43 \citep{Sivakumar2018}.

I have tried to make all software written during this thesis publicly available where possible.
This can be downloaded from the GitHub platform. 
 This includes the RNA-seq analysis pipeline used in all chapters\footnotemark , software developed to find and classify cryptic splicing (chapter 3)\footnotemark, and software written to classify splicing events found in the TDP-43 LCDmut and RRM2mut mice (chapter 5)\footnotemark.

\footnotetext[1]{\url{https://www.ncbi.nlm.nih.gov/sra}}
\footnotetext[2]{\url{https://github.com/plagnollab/RNASeq_pipeline}}
\footnotetext[3]{\url{https://github.com/jackhump/CryptEx} }
\footnotetext[4]{\url{https://github.com/jackhump/Two_TDP-43_Mutant_Mice}}

\cleardoublepage

%\pagenumbering{roman}
\setcounter{secnumdepth}{1}
\setcounter{tocdepth}{1}
%\tableofcontents %prints table of contents
%\listoffigures %prints list of figures	
%\listoftables %prints list of tables

% print lists of figures and tables without clearpage between them
\begingroup
	\let\cleardoublepage\relax  % book
	\let\clearpage\relax        % report
	\tableofcontents
	\vspace{10mm}
	\listoffigures
	\vspace{10mm}
	\listoftables
\endgroup


\clearpage



%\hspace{0pt}
%\vfill
\section*{\LARGE{List of abbreviations}}
\begin{table}[h!]
	\begin{tabular}{ll}
		ALS & Amyotrophic lateral sclerosis \\
		CLIP & UV crosslinking and immunoprecipitation \\
		eCLIP & Enhanced CLIP \\
		ES cell & Embryonic stem cell \\
		ENU & N-ethyl-N-nitrosourea \\
		FDR & False discovery rate \\
		FTD	& Frontotemporal dementia \\
		FUS & RNA-binding protein Fused in Sarcoma \\
		GO & Gene ontology \\
		GWAS & Genome-wide association study \\
		hnRNP &	Heterogeneous nuclear ribonucleoprotein \\
		iCLIP & Individual nucleotide resolution CLIP \\
		KO & Knockout \\
		LCD & Low complexity domain \\
		LCDmut & TDP-43 Low complexity domain mutation\\
		LINE & Long interspersed nuclear element \\
		MUT & Mutant \\ 
		NLS & Nuclear localisation signal \\
		NMD & Nonsense-mediated decay \\
		PSI & Percentage spliced in \\
		PTC	& Premature termination codon \\
		RBP & RNA-binding protein \\
		RNA & Ribonucleic acid \\
		RRM & RNA-recognition motif \\
		RRM2mut & TDP-43 RNA recognition motif 2 mutation \\
		RNA-seq & RNA sequencing \\
		RT-PCR & Reverse transcription polymerase chain reaction \\ 
		SINE & Short interspersed nuclear element \\
		snRNA & Small nuclear RNA \\
		snRNP & Small nuclear ribonucleoprotein \\
		SRA & Sequence read archive \\
		TDP-43 & Transactive response DNA binding protein, 43 kDa (protein) \\
		\textit{TARDBP} & Transactive response DNA binding protein (gene) \\
		TSS & transcription start site \\
		UTR & untranslated region \\
		WT & Wildtype \\
	\end{tabular}
\end{table}
\hspace{0pt}
\vfill

\pagebreak
\clearpage

\section*{Acknowledgements}

My interest in the genetics of neurological disease was kindled by my time during my undergraduate degree with James Cox in the lab of John Wood.
While there I realised the importance of computational methods and the insight they can bring to biology.  
When I began my PhD in September 2014 I knew nothing about RNA sequencing or computer programming, nor the great body of work on the molecular basis of ALS and FTD. 
That I've managed to complete a thesis on these topics is entirely down to the kindness and patience of so many brilliant scientists in both the Genetics Institute and the Institute of Neurology.  

I want to thank Warren Emmett and Kitty Lo for being my closest mentors and friends in the Plagnol lab. 
Warren in particular taught me everything I know about splicing, and quite a lot about life too.
In my office at the Darwin Building I have to thank Lucy Van Dorp, Dave Curtis, Chris Steele, Cian Murphy, Seth Jarvis, Claire Tkacz, Mike Scott, Leilei Cui, Flo Camus, Niko Pontikos and Francois Balloux for making computational biology an exciting and fun discipline to work in.

From my colleagues in the Fratta lab I must specifically thank Agnieszka Ule, Nicol Birsa and Prasanth Sivakumar for providing me with both experimental data and with friendship.
In the Institute of Neurology I worked with some wonderful people including Anny Devoy, Justin Tosh, Laura Pulford (RIP), Lizzy Fisher, Lauren Gittings, Katie Wilson, Teresa Niccoli, Nejc Haberman, Igor Ruiz De Los Mozos, Nobby Chakrabarti and Charlotte Capitanchik as well as the rest of the Fratta, Isaacs, Fisher, Ule, and Luscombe labs.
None of this work would have been possible without the fantastic administrative support of Rosie Baverstock-West, Agata Blaszczyk, Susmita Datta, James Michaels, Tristan Clark and David Gregory.

I am truly grateful for the support given to me by my three supervisors: Adrian Isaacs, Vincent Plagnol and Pietro Fratta. 
My PhD has been deeply rewarding and I feel I've really benefited from their diverse experiences and approaches, as well as occasional differences of opinion.
It's truly been a dream team of PhD supervision and I look forward to collaborating with you all in the future.

Finally, I owe a deep debt of gratitude to my friends and family for the love and support they've shown me throughout these 4 years.
\vspace{10mm}

\begingroup
\centering
\textit{For Aliss Pollock}

\textit{Without you the world would be a far duller place.}


\endgroup