
\begin{titlepage}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newcommand{\HRule}{\rule{\linewidth}{0.5mm}}

\center 

%logo
\includegraphics[width=7cm]{Figures/misc/ucllogo.jpg}\\[1cm]
%headings
\textsc{\LARGE University College London}\\[2cm] % Name of your university/college
%title
\HRule \\[1cm]
%CHANGE TITLE?
{ \huge \bfseries RNA dysregulation in models of frontotemporal dementia and amyotrophic lateral sclerosis}\\[0.4cm] 
\HRule \\[2cm]


\textsc{\Large Doctoral Thesis}\\[1.5cm] % Minor heading such as course title


%Author
%\begin{minipage}[t]{0.8\textwidth}
%\begin{flushleft} \large
%\raggedright
\emph{Author:}\\
Jack \textsc{Humphrey}\\[1cm] % Author's Name
%\end{flushleft}
%\end{minipage}

%\begin{minipage}[t]{0.8\textwidth}
%\begin{flushright} \large
%\raggedleft
\emph{Supervisors:} \\
Prof. Adrian \textsc{Isaacs}\\ 
Dr Vincent \textsc{Plagnol}\\ 
Dr Pietro \textsc{Fratta}\\[1cm]


\textsc{\Large UCL Institute of Neurology}\\[0.5cm] % Major heading such as course name
\textsc{\Large UCL Genetics Institute}\\[2cm] % Major heading such as course name


%\end{flushright}
%\end{minipage}\\[3cm]

%date
%{\large \today}\\[3cm] 

\vfill % Fill the rest of the page with whitespace

\end{titlepage}%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\cleardoublepage
%\clearpage

%\phantomsection


I confirm that the work presented in this thesis is my own.  Where information has been derived from other sources, I confirm that this has been indicated in the thesis.

Jack \textsc{Humphrey}
\vfill




\cleardoublepage

%\phantomsection
%\begin{abstract}
\section*{Abstract}
%% update! 300 words max!

Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) are two rare but devastating neurodegenerative diseases that share pathological features and genetic factors. A central problem in  both diseases is understanding the role of RNA-binding proteins, exemplified by transactive response DNA-binding protein 43kDa (TDP-43) and fused in sarcoma (FUS). These proteins play a vital role in RNA regulation in all cells but in diseased neurons they alter their function to form potentially pathogenic aggregates. These can be linked to genetic mutations in some rare cases, although most cases of ALS/FTD have no known genetic cause. My work uses the revolutionary technology of RNA sequencing to understand changes in RNA expression, splicing and transport in different cellular and animal models of sporadic and genetic disease.\\
I present a meta-analysis of published RNA sequencing experiments which deplete TDP-43 or FUS as a crude model of sporadic disease. Using these datasets, I assess the evidence for cryptic splicing, an unusual RNA phenotype caused by a loss of mRNA splicing fidelity. I demonstrate that repression of cryptic splicing is a specific function of TDP-43 and not FUS. My analysis predicts that cryptic splicing leads to expression changes due to disrupted mRNA stability and may therefore be a cause of neurodegeneration in ALS and FTD.\\ 
I then describe work on a new mouse model of FUS-mediated ALS. I observe that gene expression changes are seen only in the spinal cord and not in the cortex. These expression changes are progressive, occurring late into the lifespan of the mice and are accompanied by motor neuron loss. The altered mRNAs are enriched in mitochondrial and ribosomal genes, which warrant further investigation into the mutant protein's role in translation and transport of its target mRNAs. 
Lastly, I map out the remaining projects for the rest of my PhD. My goal is to transfer the insight gained from the model work into establishing mechanisms of disease course and aetiology in the analysis of human patient data. 






%\end{abstract}
%\clearpage

\cleardoublepage

\section*{Impact Statement}

% intro
% aims of thesis
% how these insights are useful to the field
This thesis comprises four studies on TDP-43 and FUS, two proteins linked to the devastating neurodegenerative diseases Amyotrophic Lateral Sclerosis and Frontotemporal Dementia.
I analysed RNA sequencing data to develop insights in to the functions of these two proteins in different models of disease.
This work has uncovered new roles for TDP-43 and FUS in specific types of RNA regulation.
Additionally, I have discovered new mechanisms to explain how disease-associated mutations in the two proteins affect these roles.
Conclusions from my work are applicable to the RNA biology field as a whole as well as efforts to understand and treat these diseases.
Insights into RNA-binding proteins and RNA regulation from this work can be transferred to many non-neurological diseases including muscular diseases, certain subtypes of cancer and retinal diseases \citep{Scotti2015}. 
Several of my chapters focus on developing new methods or adapting existing methods to analyse RNA splicing, a key mechanism of RNA regulation.
RNA-sequencing is becoming a standard experiment across all fields of biology and the use of sequencing data in analysis of RNA splicing is an area of extreme interest and method development. 

% impacts arising from this thesis:
Three peer-reviewed papers have been published from this thesis.
As of November 2018, "Quantitative analysis of cryptic splicing associated with TDP-43 depletion" \citep{Humphrey2017} has been cited 11 times, "Humanized mutant FUS drives progressive motor neuron degeneration without aggregation in FUS Delta14 knockin mice" \citep{Devoy2017} has been cited 11 times and "Mice with endogenous TDP-43 mutations exhibit gain of splicing function and characteristics of amyotrophic lateral sclerosis" \citep{Fratta2018} has been cited 7 times and was the subject of a "News and Views" article in The EMBO Journal \citep{Rouaux2018}.
These outcomes illustrate the importance of this work to the neurodegenerative disease and RNA biology fields.

Three new mouse models of ALS/FTD have been generated for projects carried out in this thesis: the FUS $\Delta$14 mouse, the TDP-43 RRM2mut mouse, and the TDP-43 LCDmut mouse. 
All three mice are available for labs around the world to use in further experiments.
RNA-seq data created from these mice have been made publicly available and can be downloaded from the Sequence Read Archive\footnotemark.
Several of my chapters rely on the re-analysis of published RNA sequencing data. 
I am pleased to see that sequencing data analysed in this thesis (chapter 5) has been reused in a further paper on TDP-43 \citep{Sivakumar2018}.

I have tried to make all software written during this thesis publicly available where possible.
This can be downloaded from the GitHub platform. 
 This includes the RNA-seq analysis pipeline used in all chapters\footnotemark , software developed to find and classify cryptic splicing (chapter 3)\footnotemark, and software written to classify splicing events found in the TDP-43 LCDmut and RRM2mut mice (chapter 5)\footnotemark.

\footnotetext[1]{\url{https://www.ncbi.nlm.nih.gov/sra}}
\footnotetext[2]{\url{https://github.com/plagnollab/RNASeq_pipeline}}
\footnotetext[3]{\url{https://github.com/jackhump/CryptEx} }
\footnotetext[4]{\url{https://github.com/jackhump/Two_TDP-43_Mutant_Mice}}

\cleardoublepage

%\pagenumbering{roman}
\setcounter{secnumdepth}{1}
\setcounter{tocdepth}{1}
\tableofcontents %prints table of contents
\listoffigures %prints list of figures	
\listoftables{} %prints list of tables
\clearpage



%\hspace{0pt}
%\vfill
\section*{\LARGE{List of abbreviations}}
\begin{table}[h!]
	\begin{tabular}{ll}
		ALS & Amyotrophic lateral sclerosis \\
		CLIP & UV crosslinking and immunoprecipitation \\
		eCLIP & Enhanced UV crosslinking and immunoprecipitation \\
		ES cell & Embryonic stem cell \\
		ENU & N-ethyl-N-nitrosourea \\
		FDR & False discovery rate \\
		FTD	& Frontotemporal dementia \\
		FUS & RNA-binding protein Fused in Sarcoma \\
		GO & Gene ontology \\
		hnRNP &	Heterogeneous nuclear ribonucleoprotein\\
		iCLIP & Individual nucleotide resolution UV crosslinking and immunoprecipitation \\
		KO & Knockout \\
		LCD & Low complexity domain \\
		LCDmut & TDP-43 Low complexity domain mutation\\
		LINE & Long interspersed nuclear element \\
		MUT & Mutant \\ 
		NLS & Nuclear localisation signal \\
		NMD & Nonsense-mediated decay \\
		PSI & Percentage spliced in \\
		PTC	& Premature termination codon \\
		RBP & RNA-binding protein \\
		RNA & Ribonucleic acid \\
		RRM & RNA-recognition motif \\
		RRM2mut & TDP-43 RNA recognition motif 2 mutation \\
		RNA-seq & RNA sequencing \\
		RT-PCR & Reverse transcription polymerase chain reaction \\ 
		SINE & Short interspersed nuclear element \\
		snRNA & Small nuclear RNA \\
		snRNP & Small nuclear ribonucleoprotein \\
		SRA & Sequence read archive \\
		TDP-43 & Transactive response DNA binding protein, 43 kDa (protein) \\
		\textit{TARDBP} & Transactive response DNA binding protein (gene) \\
		TSS & transcription start site \\
		UTR & untranslated region \\
		WT & Wildtype \\
	\end{tabular}
\end{table}
\hspace{0pt}
\vfill

\pagebreak
\clearpage
