% Two different strands here - Disease biology (the problem) 
% and sequencing technology (the method) to generate insight

\chapter{Introduction}

\section{Amyotrophic Lateral Sclerosis and Frontotemporal Dementia} % the disease problem

%introduce two diseases, compare and contrast
Amyotrophic Lateral Sclerosis (ALS) is a progressive neurodegenerative disorder primarily affecting the motor neurons of the cerebral cortex and the spinal cord. It affects 2-16 people per 100,000 \citep{Logroscino2010}. Patients gradually lose voluntary motor control of their limbs and the muscles involved in speaking and swallowing. Death usually occurs within 2-3 years after the first sign of symptoms, usually from infection caused by the inability to swallow. Frontotemporal Dementia is a progressive neurodegenerative disorder primarily affecting the frontal and temporal lobes. It affects 15-22 people per 100,000 and is the second most common dementia after Alzheimer's disease \citep{Onyike2013}. Depending on the subtype of FTD, patients exhibit worsening behaviour inhibition, language production or comprehension. Both disorders peak in incidence at around 60 years of age, are invariably fatal and have no cure. These two disorders are now recognised to be two ends of a continuum called ALS/FTD. This is in part due to a sharing of symptoms in some cases, as FTD patients can exhibit motor deficits and ALS patients can exhibit cognitive decline, but also due to a striking concordance in pathology and genetics. 

\subsection{Both disorders share pathology}
% pathology work - identification of TDP-43 and FUS inclusions

Both disorders have recognisable brain pathology upon autopsy, with the affected brain regions showing aggregated protein inclusions in the nucleus and cytoplasm of neurons and glia. In FTD around 35\% of patients have inclusions positive for Tau, a microtubule-associated protein encoded by the \textit{MAPT} gene also linked to Parkinson's and Alzheimer's disease \citep{Rademakers2004}. The rest of FTD patients present with ubiquitinated inclusions containing one of two proteins: TAR DNA-binding protein 43kDa (TDP-43) \citep{Neumann2006-re} or fused in sarcoma (FUS) \citep{Neumann2009}. In ALS almost all patients present with TDP-43 positive inclusions \citep{Neumann2006-re} and a small number display FUS inclusions \citep{Vance2009-ye}, firmly cementing the link between the two disorders and a key role for TDP-43 and FUS.

\subsection{Both disorders share genetics}
% Genetics - following SOD1, the identification of rare TARDBP and FUS mutations.  
The progress in understanding the pathology of ALS/FTD has been mirrored by the progress in locating causative genes. This was initially done by linkage studies, where blocks of shared genetic information were identified in the affected members of large families. \textit{SOD1} was the first gene linked to ALS in series of families over 20 years ago \citep{Rosen1993}. Mutations in MAPT were then found in a large number of familial FTD cases \citep{Hutton1998}, linking the protein pathology with alterations to the gene itself. This theme continued in the discovery a series of rare mutations in TARDBP, the gene that codes for TDP-43 in familial cohorts of ALS and FTD \citep{Sreedharan2008-xv}. This was followed by the discovery of patients carrying mutations in the FUS gene \citep{Vance2009-ye}. The last hurrah for linkage studies came in the solving of a long standing mystery. Multiple ALS and FTD pedigrees had been linked to a region on chromosome 9, which was revealed to be a large expansion in the intron of the \textit{C9orf72} gene \citep{Renton2011,DeJesus-Hernandez2011}. In individuals of caucasian ancestry the expansion is found in 5-10\% of sporadic ALS and FTD cases, 40\% of ALS and 25\% of FTD cases with a family history \citep{Majounie2012}, more than all the other known genes put together and making it the single largest genetic contribution to ALS/FTD. At the same time, the emergence of next-generation sequencing technologies has moved the gene hunting field from conducting linkage in family pedigrees to large-scale studies comparing the allele frequencies between groups of affected and unaffected people, at first in exomes (the total protein coding portion of the genome) and soon to full genomes. This has been extremely fruitful in identifying causative mutations in a wide range of genes, recently reviewed in ALS \citep{Taylor2016} and FTD \citep{Pottier2016}. Broadly, the proteins these genes code for can be grouped by their functions. \textit{OPTN}, \textit{UBQLN2}, \textit{SQSTM1}, \textit{CHMP2B} and \textit{TBK1} have all been linked to protein degradation, whereas \textit{DCTN1}, \textit{CHCHD10} and \textit{TUBA4A} have been linked to microtubule transport and stability. The third group of genes encode RNA-binding proteins, and this is the function of \textit{TARDBP} and \textit{FUS}, as well as \textit{MATR3}, \textit{TAF15}, \textit{hnRNPA1} and \textit{hnRNPA2B1} . The proteins these genes code for have been linked to splicing, transcription, translation and transport of mRNA. 

The evidence from both the pathology and the genetics together create the RNA hypothesis of ALS and FTD, where impaired RNA regulation due to mutations or mislocalisation of RNA binding proteins is progressively toxic to neurons.

%Advances in high throughput sequencing technology have enabled the finding of causative genes in both disorders, from small family pedigrees to large international cohorts. Unsurprisingly, rare mutations have been found in \textit{TARDBP} the gene coding for TDP-43 \citep{Sreedharan2008-xv}, as well as in \textit{FUS} \citep{Vance2009-ye}.
%
%
%
%
%Each year more genes are linked to ALS and FTD and a sizeable proportion of the proteins they code for bind RNA. These RNA-binding proteins  
%%RNA binding proteins MATR3, TAF15, hnRNPA1, hnRNPA2B1
%%
%%FTD has incidence of 15-22 per 100,000 \citep{Onyike2013}
%%
%%ALS has incidence of 2-16 people per 100,000 in Europe \citep{Logroscino2010}
%%

%Introduce the two diseases. Introduce the RNA regulation theory of disease but also discuss the other possibilities

%Two bad neurodegenerative diseases
%Describe clinical presentations of both
%The ALS/FTD spectrum - linked through TDP-43 and FUS
%
%Evidence for RNA processing involvement in ALS/FTD
%
%
%%ALS/FTD and RNA-binding proteins


%In the case of TDP. FUS, A1/A2, Matr3, RBPs are mutated but with C9 RBPs are sequestered by the repeat. And yet all patients have similar phenotype.


\section{Neuronal regulation of mRNA} % the biological problem
% from the paper:
%explain RNA regulation
%
%What do neurons need?
%Long genes with complex isoforms to increase protein diversity
%Need to transport RNA and protein over long distnace
%Local translation at synapses - control over protein expression

Of all the cells in the human body, neurons arguably make the largest demands upon the transcription and splicing machinery. Neuron-specific genes tend to be much longer than in other tissues \citep{Sibley2015} and an individual neuronal gene can be processed by alternate splicing to create 1000s of mRNA and subsequent protein isoforms \citep{Treutlein2014}. The distinct compartments of a neuron's architecture requires exquisite fine-tuning of protein function to suit its location, for example on either side of a synapse. There is also the matter of transport. Motor neurons can have axons over a metre long, along which an mRNA would have to travel to reach ribosomes close to a synapse for local translation. It is easy to hypothesise how small defects in splicing efficacy or mRNA transport could have catastrophic consequences for particular groups of neurons. 

%Explain splicing at a high level. The need for a high fidelity across the transcriptome and the potential for diversity in the proteome. 


\section{RNA-sequencing is a revolutionary technology to quantify RNA expression and splicing} % the new hot method

RNA sequencing, henceworth written as RNA-seq, is the application of modern high throughput sequencing to  to directly determine the sequence of input RNA molecules \citep{Wang2009}. Unlike the older microarray technology which relies on choosing a set of RNA probes to measure, RNA-seq is hypothesis-free. It is also highly sensitive and can pick up very lowly expressed genes. Instead of measuring the intensity of a probe, the abundance of a particular RNA molecule is calculated simply by counting the number of sequencing reads that contain its sequence. As sequencing technology has improved and reduced in cost, more complicated aspects of RNA regulation are now observable. Alternate splicing can be measured by the number of sequencing reads split across multiple exons: the splice junction. Complicated isoforms can be reconstructed from splice junctions where sequencing is sufficiently deep.

Multiple groups across the world have used RNA-seq to investigate gene expression and splicing changes in all manner of models of ALS/FTD. The field as a whole is progressing towards more subtle and genuinely disease-like models; from the first knock-out mice, to overexpression of human mutant proteins to the more modern knock-in models where the mutant protein is at last expressed at a physiological level. While we can be more confident that the read-outs from these experiments are closer to the the truth in humans, they suffer from a much longer generation time due to replicating diseases of old age. Changes in RNA may be subtler than we currently have the power to detect or more baroque than we can yet understand. However the real power of an RNA-seq experiment is that it is open platform. This means once the data is generated it can be used in light of whatever the most up-to-date reference genome, transcript annotations or hot hypothesis happens to be. Coupled with the requirement to make raw sequencing data publicly available, RNA-seq allows for large scale re-analyis and meta-analysis in light of new discoveries and ideas. This ease of replicability is a triumph for modern biology.

%
%Overview of RNA-sequencing and the bioinformatic tools used to assay the data
%
%Differential expression 
%
%With higher read depth and longer reads comes the reconstruction of transcripts through splice junctions
%
%Observation of splicing regulation through the creation of splice junctions



%
%\section{Genetic models of ALS-FTD} % what has been done before
%% trend towards subtler modelling of disease coupled with better quality sequencing to look for subtler effects on RNA processing
%Each chapter will discuss previous models and summarise what has been found so far.
%
%Lots of previous work on ALS-FTD linked RNA-binding proteins
%
%High level overview - models are becoming more representative of sporadic disease to unpick this
%
%
%
%



\section{Aims of my PhD} % what I plan to contribute to the field - final section

The aim of my PhD is to capitalise on the RNA-seq revolution and bring together many disparate models of ALS and FTD to fully assess the nature of RNA dysregulation in disease.
I have performed a meta-analysis of published TDP-43 depletion datasets to assess the evidence of cryptic splicing, a recently discovered RNA phenotype. I am now in the middle of analysing a series of new TDP-43 and FUS associated disease mouse models generated by the Institute of Neurology. Finally, I will then use the insight gleaned from the cellular and animal models to generate novel hypotheses to explain the mechanism of ALS/FTD using RNA-seq data from human patients.

%
%
%can I use RNA-seq to gain new insight into the real human disease from the model data?
%use existing tools and develop new tools to better understand RNA processing in ALS/FTD
%
%	