\chapter{A meta-analysis of FUS splicing comparing knockouts with patient-like $\Delta$NLS mutations}


\section{Background}


\section{Methods}

\subsection{Data processing}
Usual pipeline. Aligned with STAR, reads counting with HTSeq-count. 

\subsection{Differential Expression}
DESeq2 (citation)
Model all data together with a dataset covariate to maximise power.
Extract condition specific fold changes (Wald test). False Discovery Rate threshold of 5\% applied. Shrunken fold change values reported. Gene expression values are reported as raw counts multiplied by each samplesâ€™ size factor generated by DESeq2.

\subsection{Differential Splicing}
SGSeq (Goldstein)
Group data into conditions and model fold change per comparison. SGSeq looks for all potential splicing variants in each sample and then counts the reads supporting each the inclusion or exclusion of that splicing variant. 
Percentage Spliced In (PSI) values (Katz) for each splicing variant were calculated by taking the read counts supporting the inclusion event and dividing by the total reads in that event. 

For individual splicing events, PSI values were compared between each condition using a one-way ANOVA with post-hoc Tukey test.

\subsection{Gene Ontology}
GProfileR package (Reimand et al, NAR). GO categories were hand-curated and restricted to at least 5 genes per set. P-values reported after Bonferroni correction. 

\subsection{Polyadenylation and iCLIP}
PolyAsite mm10 v1.0 (Zavolan group - Gruber et al Genome Res.). Each site must be supported by at least 2 datasets.
iCLIP data from Rogelj via iCOUNT .

\subsection{Conservation}
Per nucleotide PhyloP conservation scores (Pollard) comparing mouse (mm10) with 60 ??? animals was downloaded from UCSC. For each splicing variant the entire length of the encompassing intron was taken and the median PhyloP score calculated.

\clearpage


\begin{longtable}[]{@{}llllll@{}}
	\toprule
	\begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Dataset}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Tissue}}\strut
	\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
		{\textbf{Controls}}\strut
	\end{minipage} & \begin{minipage}[t]{0.10\columnwidth}\raggedright\strut
		{\textbf{Age}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Knockout (KO)}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Mutation (MUT)}}\strut
	\end{minipage}\tabularnewline\hline
	\begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{\textbf{Dupuis}}
		{\footnotesize\citep{Scekic-zahirovic2016}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{Whole brain}\strut
	\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
		{Separate}\strut
	\end{minipage} & \begin{minipage}[t]{0.10\columnwidth}\raggedright\strut
		{E18.5}\strut
	\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{Gene trap in intron 1}\strut
	\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{Stop codon after exon 14 ($\Delta$NLS)}\strut
	\end{minipage}\tabularnewline\hline
	\begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{\textbf{Bozzoni}}
		{\footnotesize\citep{Capauto2018}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{Motor neurons}
		
		{cultured from mESCs}\strut
	\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
		{Shared}\strut
	\end{minipage} & \begin{minipage}[t]{0.10\columnwidth}\raggedright\strut
		{-}\strut
	\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{Gene trap in exon 12}
		
		{ \footnotesize\citep{Hicks2000} }\strut
	\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{P517L knock-in,}
		{corresponding to human P525L}
		{\footnotesize\citep{Conte2012}}\strut
	\end{minipage}\tabularnewline\hline
	\begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{\textbf{Fratta}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{Spinal cord}\strut
	\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
		{Separate}\strut
	\end{minipage} & \begin{minipage}[t]{0.10\columnwidth}\raggedright\strut
		{E18.5}\strut
	\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{Gene trap in intron 1}\strut
	\end{minipage} & \begin{minipage}[t]{0.16\columnwidth}\raggedright\strut
		{Delta exon 14 }
		{\footnotesize\citep{Devoy2017}}  \strut
	\end{minipage}\tabularnewline
	\bottomrule
	\caption{The three FUS mouse datasets}
	\label{tab:fus_datasets}
\end{longtable}

% SEQUENCING STATS

\begin{longtable}[]{@{}llllll@{}}
	\toprule
	\begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Dataset}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Replicates per condition}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Library type}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Mapped reads (millions)}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Read type}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{SRA accession}}\strut
	\end{minipage}\tabularnewline\hline
	\begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Dupuis}}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{4-5}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{mRNA}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{15-25}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{1 x 50bp}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{SRP070906 }\strut
	\end{minipage}\tabularnewline
	\begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Bozzoni} }\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{3}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{Total RNA}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{34-52}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{2 x 100bp}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{SRP111475}\strut
	\end{minipage}\tabularnewline
	\begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{\textbf{Fratta} }\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{4}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{Total RNA}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{52-65}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{2 x 150bp}\strut
	\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
		{-}\strut
	\end{minipage}\tabularnewline
	\bottomrule
	\caption{RNA-seq statistics of the three datasets}
	\label{tab:fus_sequencing}
\end{longtable}



\section{Results}

\subsection{Synaptic and RNA-binding genes are a common gene expression response to FUS nuclear depletion}
We combined all 3 datasets to maximise power to discover differentially expressed genes. 

Simply overlapping the genes that were significant at FDR < 0.05 resulted in an overlapping 425 genes
Instead taking genes that were significant at FDR < 0.05 in one comparison and weakly significant at P < 0.05 increased the overlap to 1300 genes,  meaning that the majority of differentially expressed genes are shared between Fus knockout and Fus mutation.

Estimated fold changes between knockout and mutation were highly concordant in the overlapping genes with only 5 genes responding in opposite directions between conditions. Fitting a linear regression between the estimated fold changes in knockout and mutation showed a clear bias towards stronger fold changes in knockout than mutation. 
This suggests that the mutating FUS leads to a loss of function phenotype that is less extreme than knocking out the gene.

Fus itself was downregulated in knockouts specifically. Its estimated log2 fold change appears to be conservatively high (-0.6, equivalent to to a 35\% reduction) is partly due to the Bayesian shrinkage of all fold changes and as well to to the relatively poor knockout seen in the Bozzoni data. Trove2, a gene encoding for the 60 kDa SS-A/Ro ribonucleoprotein was specifically downregulated in Fus mutants and not in knockouts, whereas the fellow FET family RNA binding protein Taf15 was reliably upregulated in both conditions. Taf15 upregulation has been previously suggested by Dupuis as a cellular response to Fus depletion due to their strong overlap in RNA targets (Yeo paper). Strikingly we observed the strongest upregulation in a set of X chromosome genes, Xlr3a, Xlr4a, Xlr4b. We were initially concerned that this was driven by a sex imbalance in our samples as embryonic mice are not routinely sexed. Imputing genetic sex through Y chromosome gene expression suggested that although some of the samples are unbalanced in sex between conditions the sex differences do not explain the upregulation of Xlr genes.

Gene ontology (GO) terms enriched in the overlapping genes were strongly direction specific, with terms involving RNA binding, splicing and metabolism enriched in upregulated genes whereas synaptic and neuronal terms were enriched in downregulated genes.

Knockout- and mutation- specific genes were less clearly enriched in specific functions. Knockout specific genes were involved in extracellular membrane functions, ion channels and amino acid transport whereas the 186 mutant specific genes showed an enrichment in Dopaminergic synapses and metabolism.

% TABLE OF GENE EXPRESSION NUMBERS
\begin{table}[h!]
	%\begin{centerline}
		\begin{tabular}{|r|ccc|ccc|}
			\hline
			& Bozzoni & Dupuis & Fratta & Bozzoni & Dupuis & Fratta\\
			& MUT & MUT & MUT & KO & KO & KO\\
			\hline
			Individual hits                & 19 & 1552 & 88 & 100 & 2916 & 151 \\
			Overlapping joint model & 5 & 368 & 57 & 51 & 1007 & 114 \\
			Unique to dataset          & 14 & 1184 & 31 & 49 & 1909 & 37 \\
			\hline
			\textbf{Joint model}       & \multicolumn{3}{c|}{754} & \multicolumn{3}{c|}{2136} \\
			\hline
			Overlap (strict)              & \multicolumn{2}{c}{329} & \multicolumn{2}{|c|}{\textbf{425}} & \multicolumn{2}{c|}{1711} \\
			Overlap (relaxed)           & \multicolumn{2}{c}{186} & \multicolumn{2}{|c|}{\textbf{1308} } & \multicolumn{2}{c|}{961} \\
			\hline
		\end{tabular}
	%\end{centerline}
	\caption{Results from separate and joint differential expression analysis}
	\label{tab:expression_results}
\end{table}




\subsection{FUS modulates the inclusion of a set of highly conserved RNA-binding protein introns}

\begin{table}[h!]
	%\begin{centerline}
		\begin{tabular}{|r|ccc|ccc|}
			\hline
			%\centering
			& Bozzoni & Dupuis & Fratta & Bozzoni & Dupuis & Fratta\\
			& MUT & MUT & MUT & KO & KO & KO\\
			\hline
			Individual hits                & 31 & 1 & 56 & 211 & 46 & 230 \\
			Overlapping joint model & 7 & 1 & 30 & 143 & 38 & 169 \\
			Unique to dataset          & 24 & 0 & 26 & 68 & 8 & 61 \\
			\hline
			\textbf{Joint model}       & \multicolumn{3}{c|}{93} & \multicolumn{3}{c|}{890} \\
			\hline
			Overlap (strict)              & \multicolumn{2}{c}{33} & \multicolumn{2}{|c|}{\textbf{60}} & \multicolumn{2}{c|}{830} \\
			Overlap (relaxed)           & \multicolumn{2}{c}{16} & \multicolumn{2}{|c|}{\textbf{405} } & \multicolumn{2}{c|}{501} \\
			\hline
		\end{tabular}
	%\end{centerline}
	\caption{Results from separate and joint splicing analysis}
	\label{tab:splicing_results}
\end{table}

\subsection{FUS autoregulation is dependent on intron retention}



\section{Discussion}